src/test/java/org/kiwiproject/dropwizard/error/dao/jdk/NoOpApplicationErrorDaoTest.java

```diff
@@ -0,0 +1,121 @@
+package org.kiwiproject.dropwizard.error.dao.jdk;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.kiwiproject.dropwizard.error.dao.ApplicationErrorStatus;
+import org.kiwiproject.dropwizard.error.model.ApplicationError;
+import org.kiwiproject.test.junit.jupiter.params.provider.RandomLongSource;
+
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+@DisplayName("NoOpApplicationErrorDao")
+class NoOpApplicationErrorDaoTest {
+
+    private NoOpApplicationErrorDao errorDao;
```

Comment:
should this be `var` or the interface type?

Reply:
You can't declare a field with var. Since this is a test I don't have any issue declaring it as the concrete type.

Reply:
ok